README.md

```diff
@@ -84,29 +84,56 @@ This will add the specific versions of the releases that are currently deployed
 }
 ```
 
-### Deploy to local envirionment
+### Generate a manifest to only deploy certain packages
+
+```
+python .\generate_manifest.py -p Huddle.XYZ
+```
+
+This will only add the specified project to the manifest:
+
+```
+{
+    'Projects':
+    {
+        'Huddle.XYZ':
+            {
+                'Packages': ['Huddle.XYZ1', 'Huddle.XYZ2']
+            }
+    },
+    'StagingLocation': 'D:\\dev\\huddle\\StagingLocation'
+}
+```
+
+### Deploy to local environment
 
 Deploying to a local environment helps set up developers with the latest code or reproduce a given environment for debugging on your developer workstation.
 
-It reads in the `manifest.json` file that describes the state of the local environment.
+It reads in the manifest file supplied that describes the state of the local environment.
 
 ```
-python .\octopose.py
+python .\octopose.py .\manifest.json

```

Comment:
Or is it `cat .\manifest.json | python .\octopose.py`? or both?

Reply:
Going to support both (because most unix style tools do that), but had to do some wrangling with the encoding to get it to work both ways.

Reply:
👍